app/components/subject-picker/template.hbs

```diff
@@ -2,9 +2,8 @@
     {{#each selected as |subject|}}
         <span class="subject">
             {{#each subject as |segment|}}
-                <b>{{segment.text}}</b>
+                <div class="subject-container"><b>{{segment.text}} <span class="cancel-subject" {{action 'deselect' subject}}></span></b></div>
```

Comment:
This span appears to be a font-awesome icon. Since the only sign of an interactive element is pictorial, how will a blind user accessing the site via screenreader know that there is a clickable icon/button here? (see [FA accessibility guidelines](http://fontawesome.io/accessibility/) for recommendations?

Tips include:
1. Provide aria labels
2. Use markup semantically. Generally, avoid using `span` for something clickable, because to HTML, `span` just means that there is text there. Prefer `button` for something interactive, or `a` for something that triggers navigation to a different page. (if you use a button or a link, the mouse cursor changes will also change without you needing to write extra CSS, which is nice)

In combination, screen readers like Mac OS VoiceOver use these cues to help blind users understand a page. If you ever want, we can arrange a demo to help show how the page looks from that perspective; just let me know.